metadata-ingestion/src/datahub/ingestion/source/aws/glue.py

```diff
@@ -944,21 +948,22 @@ def get_profile_if_enabled(
                 partitions = response["Partitions"]
                 partition_keys = [k["Name"] for k in partition_keys]
 
-                for p in partitions:
-                    table_stats = p.get("Parameters", {})
-                    column_stats = p["StorageDescriptor"]["Columns"]
-
-                    # only support single partition key
-                    partition_spec = str({partition_keys[0]: p["Values"][0]})
-
-                    if self.source_config.profiling.partition_patterns.allowed(
-                        partition_spec
-                    ):
-                        yield self._create_profile_mcp(
-                            mce, table_stats, column_stats, partition_spec
-                        ).as_workunit()
-                    else:
-                        continue
+                with ThreadPoolExecutor(
```

Comment:
Do we make any API call or query as part of `_create_partition_profile_mcp` ? I didn't notice any. If we are not making any network calls, then `max_workers` config does not make sense and we should remove it from here.

Reply:
@mayurinehate yes, we are not making any network calls i ll remove it.